EditPortfolio.js

```diff
@@ -0,0 +1,37 @@
+const addExperienceBtn = document.getElementById("addExperienceBtn");
+const experienceForm = document.getElementById("experienceForm");
+const experienceList = document.getElementById("experienceList");
+const experienceEntryForm = document.getElementById("experienceEntryForm");
+
+addExperienceBtn.addEventListener("click", () => {
+  experienceForm.style.display = "block";
+});
+
+experienceEntryForm.addEventListener("submit", (event) => {
```

Comment:
Why are you using eventListener here. You can also directly use a function that is passed in the onSubmit of form.

Reply:
yes, we can also use the "onSubmit" attribute but eventListener provides more flexibility and control over what happens when the form is submitted, in future tasks we would need more dynamic modifications on form data. and this approach is best for dynamic modifications that help to keep the js code more organized and easier to maintain.